### DISCUSSION TITLE
How do I obtain the version number from a different URL?

### DISCUSSION THREAD

**Discussion by hellodword** [ImranR98/Obtainium#1811] [Q&A]
The direct APK links are:

- https://github.com/FongMi/Release/blob/f0b73fc2982ccf7f45bdd9d28d3928e2b2f548f4/apk/release/leanback-python-armeabi_v7a.apk
- https://github.com/FongMi/Release/blob/f0b73fc2982ccf7f45bdd9d28d3928e2b2f548f4/apk/release/leanback-python-arm64_v8a.apk

The version info can be found here:

- https://github.com/FongMi/Release/blob/f0b73fc2982ccf7f45bdd9d28d3928e2b2f548f4/apk/release/leanback.json#L3

The app should only be marked as a new version when the version is updated.

Any suggestions?

**Comment by DarkCrypt** [comment]
Not possible with Obtainium.

**Reply by DwainZwerg** [reply]
@hellodword You could open an new issue.
